Version 0.26.0 throws exception "Could not find installation home path"

After installing the highly anticipated new release 0.26.0 of the plugin, ktlint does not work anymore at all here. :-(
If I trigger "Format with ktlint" action, I get this error in the IntelliJ log:
```plain
2025-01-08 17:30:21,118 [1741885]   INFO - STDERR - ERROR:                An error occurred while processing file '....kt':
2025-01-08 17:30:21,118 [1741885]   INFO - STDERR -                java.lang.RuntimeException: Could not find installation home path. Please make sure bin/idea.properties is present in the installation directory.
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:98)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:69)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.fromFile(BuildNumber.java:259)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy._get_value_$lambda$1$lambda$0(SynchronizedClearableLazy.kt:41)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at java.base/java.util.concurrent.atomic.AtomicReference.updateAndGet(AtomicReference.java:210)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.getValue(SynchronizedClearableLazy.kt:40)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.get(SynchronizedClearableLazy.kt:28)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.currentVersion(BuildNumber.java:302)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginManagerCore.getBuildNumber(PluginManagerCore.kt:411)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext.getDefaultVersion(DescriptorListLoadingContext.kt:53)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.IdeaPluginDescriptorImpl.readExternal(IdeaPluginDescriptorImpl.kt:190)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadDescriptorFromJar(PluginDescriptorLoader.kt:161)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader$loadForCoreEnv$1.invokeSuspend(PluginDescriptorLoader.kt:75)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadForCoreEnv(PluginDescriptorLoader.kt:74)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.com.intellij.core.CoreApplicationEnvironment.registerExtensionPointAndExtensions(CoreApplicationEnvironment.java:234)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.registerApplicationExtensionPointsAndExtensionsFrom(KotlinCoreEnvironment.kt:705)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createApplicationEnvironment(KotlinCoreEnvironment.kt:675)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironment(KotlinCoreEnvironment.kt:558)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironmentForProduction(KotlinCoreEnvironment.kt:534)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createForProduction(KotlinCoreEnvironment.kt:466)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryKt.initPsiFileFactory(KotlinPsiFileFactory.kt:58)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryProvider.getKotlinPsiFileFactory(KotlinPsiFileFactory.kt:32)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.RuleExecutionContext$Companion.createRuleExecutionContext$ktlint_rule_engine(RuleExecutionContext.kt:212)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:51)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:28)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format$default(CodeFormatter.kt:20)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format(KtLintRuleEngine.kt:145)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format$default(KtLintRuleEngine.kt:139)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.nbadal.ktlint.KtlintFormatKt.executeKtlint(KtlintFormat.kt:118)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$lambda$0(KtlintFormat.kt:51)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:341)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:147)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction$lambda$5(AnyThreadWriteThreadingSupport.kt:379)
2025-01-08 17:30:21,119 [1741886]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:389)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:379)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:896)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:145)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:226)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:188)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:154)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:121)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:341)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:329)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat(KtlintFormat.kt:50)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.nbadal.ktlint.actions.FormatAction$KtlintFormatContentIterator.processFile(FormatAction.kt:99)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.vfs.VfsUtilCore$1.visitFileEx(VfsUtilCore.java:288)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:303)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.vfs.VfsUtilCore.iterateChildrenRecursively(VfsUtilCore.java:284)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.vfs.VfsUtilCore.iterateChildrenRecursively(VfsUtilCore.java:277)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.nbadal.ktlint.actions.FormatAction.actionPerformed(FormatAction.kt:43)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.kt:374)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.ide.actions.GotoActionAction.lambda$performActionImpl$4(GotoActionAction.java:89)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.performWithActionCallbacks(ActionManagerImpl.kt:1173)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.kt:396)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.ide.actions.GotoActionAction.performActionImpl(GotoActionAction.java:88)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:68)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:236)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:198)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread$lambda$2(AnyThreadWriteThreadingSupport.kt:217)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread(AnyThreadWriteThreadingSupport.kt:216)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:842)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:421)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:107)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:101)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:27)
2025-01-08 17:30:21,120 [1741887]   INFO - STDERR - at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:117)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:675)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:573)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:857)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16(IdeEventQueue.kt:354)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:1045)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:916)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:1045)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18(IdeEventQueue.kt:349)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:395)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
2025-01-08 17:30:21,121 [1741888]   INFO - STDERR - at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
2025-01-08 17:30:21,122 [1741889]   INFO - STDERR -
2025-01-08 17:30:21,125 [1741892] SEVERE - #c.i.o.a.i.FlushQueue -                An error occurred while processing file '....kt':
               java.lang.RuntimeException: Could not find installation home path. Please make sure bin/idea.properties is present in the installation directory.
at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:98)
at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:69)
at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.fromFile(BuildNumber.java:259)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy._get_value_$lambda$1$lambda$0(SynchronizedClearableLazy.kt:41)
at java.base/java.util.concurrent.atomic.AtomicReference.updateAndGet(AtomicReference.java:210)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.getValue(SynchronizedClearableLazy.kt:40)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.get(SynchronizedClearableLazy.kt:28)
at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.currentVersion(BuildNumber.java:302)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginManagerCore.getBuildNumber(PluginManagerCore.kt:411)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext.getDefaultVersion(DescriptorListLoadingContext.kt:53)
at org.jetbrains.kotlin.com.intellij.ide.plugins.IdeaPluginDescriptorImpl.readExternal(IdeaPluginDescriptorImpl.kt:190)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadDescriptorFromJar(PluginDescriptorLoader.kt:161)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader$loadForCoreEnv$1.invokeSuspend(PuginDescriptorLoader.kt:75)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadForCoreEnv(PluginDescriptorLoader.kt:74)
at org.jetbrains.kotlin.com.intellij.core.CoreApplicationEnvironment.registerExtensionPointAndExtensions(CoreApplicationEnvironment.java:234)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.registerApplicationExtensionPointsAndExtensionsFrom(KotlinCoreEnvironment.kt:705)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createApplicationEnvironment(KotlinCoreEnvironment.kt:675)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironment(KotlinCoreEnvironment.kt:558)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironmentForProduction(KotlinCoreEnvironment.kt:534)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createForProduction(KotlinCoreEnvironment.kt:466)
at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryKt.initPsiFileFactory(KotlinPsiFileFactory.kt:58)
at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryProvider.getKotlinPsiFileFactory(KotlinPsiFileFactory.kt:32)
at com.pinterest.ktlint.rule.engine.internal.RuleExecutionContext$Companion.createRuleExecutionContext$ktlint_rule_engine(RuleExecutionContext.kt:212)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:51)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:28)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format$default(CodeFormatter.kt:20)
at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format(KtLintRuleEngine.kt:145)
at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format$default(KtLintRuleEngine.kt:139)
at com.nbadal.ktlint.KtlintFormatKt.executeKtlint(KtlintFormat.kt:118)
at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$lambda$0(KtlintFormat.kt:51)
at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:341)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:147)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction$lambda$5(AnyThreadWriteThreadingSupport.kt:379)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:389)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:379)
at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:896)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:145)
at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:226)
at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:188)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:154)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:121)
at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:341)
at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:329)
at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat(KtlintFormat.kt:50)
at com.nbadal.ktlint.actions.FormatAction$KtlintFormatContentIterator.processFile(FormatAction.kt:99)
at com.intellij.openapi.vfs.VfsUtilCore$1.visitFileEx(VfsUtilCore.java:288)
at com.intellij.openapi.vfs.VfsUtilCore.visitChildrenRecursively(VfsUtilCore.java:303)
at com.intellij.openapi.vfs.VfsUtilCore.iterateChildrenRecursively(VfsUtilCore.java:284)
at com.intellij.openapi.vfs.VfsUtilCore.iterateChildrenRecursively(VfsUtilCore.java:277)
at com.nbadal.ktlint.actions.FormatAction.actionPerformed(FormatAction.kt:43)
at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.kt:374)
at com.intellij.ide.actions.GotoActionAction.lambda$performActionImpl$4(GotoActionAction.java:89)
at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.performWithActionCallbacks(ActionManagerImpl.kt:1173)
at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.kt:396)
at com.intellij.ide.actions.GotoActionAction.performActionImpl(GotoActionAction.java:88)
at com.intellij.ide.actions.GotoActionAction.lambda$performAction$2(GotoActionAction.java:68)
at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:236)
at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:198)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread$lambda$2(AnyThreadWriteThreadingSupport.kt:217)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread(AnyThreadWriteThreadingSupport.kt:216)
at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:842)
at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:421)
at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:107)
at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:101)
at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:27)
at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:117)
at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:675)
at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:573)
at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:857)
at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16(IdeEventQueue.kt:354)
at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:1045)
at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:916)
at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:1045)
at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18(IdeEventQueue.kt:349)
at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:395)
at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

While doing an unrelated change on my local machine, I got an different exception. But it seems to have the exact same cause. I hope this will give me more insights in the problem for now:

```
java.lang.AssertionError:                An error occurred while processing file '/Users/pdingemans/git/ktlint/ktlint-examples/src/main/kotlin/Foo.kt':
               java.lang.RuntimeException: Could not find installation home path. Please reinstall the software.
at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:98)
at org.jetbrains.kotlin.com.intellij.openapi.application.PathManager.getHomePath(PathManager.java:69)
at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.fromFile(BuildNumber.java:259)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy._get_value_$lambda$1$lambda$0(SynchronizedClearableLazy.kt:41)
at java.base/java.util.concurrent.atomic.AtomicReference.updateAndGet(AtomicReference.java:210)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.getValue(SynchronizedClearableLazy.kt:40)
at org.jetbrains.kotlin.com.intellij.util.concurrency.SynchronizedClearableLazy.get(SynchronizedClearableLazy.kt:28)
at org.jetbrains.kotlin.com.intellij.openapi.util.BuildNumber.currentVersion(BuildNumber.java:302)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginManagerCore.getBuildNumber(PluginManagerCore.kt:411)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext$1.invoke(DescriptorListLoadingContext.kt:25)
at org.jetbrains.kotlin.com.intellij.ide.plugins.DescriptorListLoadingContext.getDefaultVersion(DescriptorListLoadingContext.kt:53)
at org.jetbrains.kotlin.com.intellij.ide.plugins.IdeaPluginDescriptorImpl.readExternal(IdeaPluginDescriptorImpl.kt:190)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadDescriptorFromJar(PluginDescriptorLoader.kt:161)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader$loadForCoreEnv$1.invokeSuspend(PluginDescriptorLoader.kt:75)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith$$$capture(ContinuationImpl.kt:33)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt)
at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
at org.jetbrains.kotlin.com.intellij.ide.plugins.PluginDescriptorLoader.loadForCoreEnv(PluginDescriptorLoader.kt:74)
at org.jetbrains.kotlin.com.intellij.core.CoreApplicationEnvironment.registerExtensionPointAndExtensions(CoreApplicationEnvironment.java:234)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.registerApplicationExtensionPointsAndExtensionsFrom(KotlinCoreEnvironment.kt:705)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createApplicationEnvironment(KotlinCoreEnvironment.kt:675)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironment(KotlinCoreEnvironment.kt:558)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.getOrCreateApplicationEnvironmentForProduction(KotlinCoreEnvironment.kt:534)
at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createForProduction(KotlinCoreEnvironment.kt:466)
at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryKt.initPsiFileFactory(KotlinPsiFileFactory.kt:58)
at com.pinterest.ktlint.rule.engine.internal.KotlinPsiFileFactoryProvider.getKotlinPsiFileFactory(KotlinPsiFileFactory.kt:32)
at com.pinterest.ktlint.rule.engine.internal.RuleExecutionContext$Companion.createRuleExecutionContext$ktlint_rule_engine(RuleExecutionContext.kt:212)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:51)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format(CodeFormatter.kt:28)
at com.pinterest.ktlint.rule.engine.internal.CodeFormatter.format$default(CodeFormatter.kt:20)
at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format(KtLintRuleEngine.kt:145)
at com.pinterest.ktlint.rule.engine.api.KtLintRuleEngine.format$default(KtLintRuleEngine.kt:139)
at com.nbadal.ktlint.KtlintFormatKt.executeKtlint(KtlintFormat.kt:118)
at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$lambda$0(KtlintFormat.kt:51)
at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:338)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:144)
at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:975)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:142)
at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:225)
at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:187)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:151)
at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:118)
at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:338)
at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:326)
at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat(KtlintFormat.kt:50)
at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$default(KtlintFormat.kt:35)
at com.nbadal.ktlint.KtlintSettingsComponent.apply(KtlintSettingsComponent.kt:273)
at com.nbadal.ktlint.KtlintConfig.apply(KtlintConfig.kt:22)
at com.intellij.openapi.options.ex.ConfigurableWrapper.apply(ConfigurableWrapper.java:191)
at com.intellij.openapi.options.newEditor.ConfigurableEditor.apply(ConfigurableEditor.java:311)
at com.intellij.openapi.options.newEditor.SettingsEditor$5.apply(SettingsEditor.java:200)
at com.intellij.openapi.options.newEditor.SettingsEditor.apply(SettingsEditor.java:431)
at com.intellij.openapi.options.newEditor.SettingsDialog.applyAndClose(SettingsDialog.java:204)
at com.intellij.openapi.options.newEditor.SettingsDialog.doOKAction(SettingsDialog.java:196)
at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1936)
at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1887)
at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2313)
at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
at java.desktop/java.awt.Component.processMouseEvent(Component.java:6657)
at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
at java.desktop/java.awt.Component.processEvent(Component.java:6422)
at java.desktop/java.awt.Container.processEvent(Container.java:2266)
at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4969)
at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4583)
at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4524)
at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2809)
at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:794)
at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:766)
at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:763)
at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:635)
at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:580)
at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:75)
at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:67)
at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:580)
at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:843)
at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
at java.desktop/java.awt.Dialog.lambda$show$2(Dialog.java:1081)
at java.desktop/sun.awt.SunToolkit.performOnMainThreadIfNeeded(SunToolkit.java:2170)
at java.desktop/java.awt.Dialog.show(Dialog.java:1041)
at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:875)
at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:474)
at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1745)
at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1694)
at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:104)
at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:61)
at com.intellij.ui.mac.MacOSApplicationProviderKt$initMacApplication$2$1.invoke(MacOSApplicationProvider.kt:64)
at com.intellij.ui.mac.MacOSApplicationProviderKt$initMacApplication$2$1.invoke(MacOSApplicationProvider.kt:63)
at com.intellij.ui.mac.MacOSApplicationProviderKt$submit$2.invokeSuspend(MacOSApplicationProvider.kt:164)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith$$$capture(ContinuationImpl.kt:33)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt)
at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
at com.intellij.openapi.application.impl.DispatchedRunnable.run(DispatchedRunnable.kt:43)
at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:843)
at com.intellij.openapi.application.impl.ApplicationImpl$4.run(ApplicationImpl.java:463)
at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:75)
at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:67)
at com.intellij.openapi.application.impl.ApplicationImpl.runWithImplicitRead(ApplicationImpl.java:1436)
at com.intellij.openapi.application.impl.FlushQueue.doRun$$$capture(FlushQueue.java:82)
at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java)
at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:124)
at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:44)
at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:318)
at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:589)
at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:44)
at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:589)
at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:848)
at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

	at com.intellij.openapi.diagnostic.DefaultLogger.error(DefaultLogger.java:83)
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:423)
	at com.nbadal.ktlint.KtlintLogger.error(KtlintLogger.kt:34)
	at com.nbadal.ktlint.KtlintLogger.error$default(KtlintLogger.kt:30)
	at com.nbadal.ktlint.KtlintNotifier.notify(KtlintNotifier.kt:56)
	at com.nbadal.ktlint.KtlintNotifier.notifyError(KtlintNotifier.kt:25)
	at com.nbadal.ktlint.KtlintNotifier.notifyError$default(KtlintNotifier.kt:20)
	at com.nbadal.ktlint.KtlintFormatKt.executeKtlint(KtlintFormat.kt:204)
	at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$lambda$0(KtlintFormat.kt:51)
	at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:144)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:975)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:142)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:225)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:187)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:151)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:118)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:326)
	at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat(KtlintFormat.kt:50)
	at com.nbadal.ktlint.KtlintFormatKt.ktlintFormat$default(KtlintFormat.kt:35)
	at com.nbadal.ktlint.KtlintSettingsComponent.apply(KtlintSettingsComponent.kt:273)
	at com.nbadal.ktlint.KtlintConfig.apply(KtlintConfig.kt:22)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.apply(ConfigurableWrapper.java:191)
	at com.intellij.openapi.options.newEditor.ConfigurableEditor.apply(ConfigurableEditor.java:311)
	at com.intellij.openapi.options.newEditor.SettingsEditor$5.apply(SettingsEditor.java:200)
	at com.intellij.openapi.options.newEditor.SettingsEditor.apply(SettingsEditor.java:431)
	at com.intellij.openapi.options.newEditor.SettingsDialog.applyAndClose(SettingsDialog.java:204)
	at com.intellij.openapi.options.newEditor.SettingsDialog.doOKAction(SettingsDialog.java:196)
	at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1936)
	at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1887)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2313)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6657)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at java.desktop/java.awt.Component.processEvent(Component.java:6422)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4969)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4583)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4524)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2809)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:794)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:766)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:763)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:635)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:580)
	at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:75)
	at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:67)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:580)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:843)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
	at java.desktop/java.awt.Dialog.lambda$show$2(Dialog.java:1081)
	at java.desktop/sun.awt.SunToolkit.performOnMainThreadIfNeeded(SunToolkit.java:2170)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1041)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:875)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:474)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1745)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1694)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:104)
	at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:61)
	at com.intellij.ui.mac.MacOSApplicationProviderKt$initMacApplication$2$1.invoke(MacOSApplicationProvider.kt:64)
	at com.intellij.ui.mac.MacOSApplicationProviderKt$initMacApplication$2$1.invoke(MacOSApplicationProvider.kt:63)
	at com.intellij.ui.mac.MacOSApplicationProviderKt$submit$2.invokeSuspend(MacOSApplicationProvider.kt:164)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith$$$capture(ContinuationImpl.kt:33)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt)
	at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
	at kotlinx.coroutines.debug.internal.DebugProbesImpl$CoroutineOwner.<init>(DebugProbesImpl.kt:535)
	at kotlinx.coroutines.debug.internal.DebugProbesImpl.createOwner(DebugProbesImpl.kt:514)
	at kotlinx.coroutines.debug.internal.DebugProbesImpl.probeCoroutineCreated$kotlinx_coroutines_core(DebugProbesImpl.kt:501)
	at kotlin.coroutines.jvm.internal.DebugProbesKt.probeCoroutineCreated(DebugProbes.kt:10)
	at kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt.createCoroutineUnintercepted(IntrinsicsJvm.kt:161)
	at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable(Cancellable.kt:30)
	at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable$default(Cancellable.kt:25)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:110)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:126)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:56)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch$default(Builders.common.kt:47)
	at kotlinx.coroutines.BuildersKt.launch$default(Unknown Source)
	at com.intellij.ui.mac.MacOSApplicationProviderKt.submit(MacOSApplicationProvider.kt:161)
	at com.intellij.ui.mac.MacOSApplicationProviderKt.initMacApplication$lambda$1(MacOSApplicationProvider.kt:63)
	at java.desktop/com.apple.eawt._AppEventHandler$_PreferencesDispatcher.performUsing(_AppEventHandler.java:310)
	at java.desktop/com.apple.eawt._AppEventHandler$_PreferencesDispatcher.performUsing(_AppEventHandler.java:301)
	at java.desktop/com.apple.eawt._AppEventHandler$_AppEventDispatcher$1.run(_AppEventHandler.java:569)
	at java.desktop/java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:318)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java)
	at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)
	at java.desktop/java.awt.event.InvocationEvent.<init>(InvocationEvent.java:291)
	at java.desktop/java.awt.event.InvocationEvent.<init>(InvocationEvent.java:217)
	at java.desktop/sun.awt.PeerEvent.<init>(PeerEvent.java:45)
	at java.desktop/sun.awt.PeerEvent.<init>(PeerEvent.java:40)
	at java.desktop/sun.awt.SunToolkit.invokeLaterOnAppContext(SunToolkit.java:606)
	at java.desktop/com.apple.eawt._AppEventHandler$_AppEventDispatcher.dispatch(_AppEventHandler.java:567)
	at java.desktop/com.apple.eawt._AppEventHandler.handleNativeNotification(_AppEventHandler.java:230)
```

--- a/ktlint-lib/build.gradle.kts
+++ b/ktlint-lib/build.gradle.kts
@@ -71,29 +71,13 @@ tasks {
         // Expose all ruleset implementations:
         mergeServiceFiles()
 
-        // Relocate to prevent conflicts with same packages provided by Intellij IDEA as well. The embeddable Kotlin compiler in the Ktlint
-        // jar differs from the compiler provided in the IDEA.
+        // Ktlint contains the embeddable Kotlin compiler. The Kotlin compiler brings in some packages from "org.jetbrains" and
+        // "org.intellij" which might conflict with the same packages provided by the Intellij IDEA Runtime environment. To avoid possible
+        // conflicts, the packages are shadowed so that ktlint can still use the embedded compiler, independently of the compiler which is
+        // provided in the Intellij IDEA runtime.
         // IMPORTANT: Third party suppliers of rule set need to add those relocations as well!
-        relocate("org.jetbrains.kotlin.psi.KtPsiFactory", "shadow.org.jetbrains.kotlin.psi.KtPsiFactory")
-        relocate("org.jetbrains.kotlin.psi.psiUtil", "shadow.org.jetbrains.kotlin.psi.psiUtil")
-
-        // From "compatability verification" (plugin verifier) results:
-        //     The plugin distribution bundles IDE packages
-        //       'org.jetbrains.org.objectweb.asm.signature',
-        //       'org.jetbrains.org.objectweb.asm.commons',
-        //       'org.jetbrains.org.objectweb.asm',
-        //       'org.jetbrains.org.objectweb.asm.util',
-        //       'org.jetbrains.org.objectweb',
-        //       'org.jetbrains.org.objectweb.asm.tree.analysis',
-        //       'org.jetbrains.concurrency',
-        //       'org.jetbrains.org.objectweb.asm.tree',
-        //       'org.jetbrains.org'.
-        //       Bundling IDE packages is considered bad practice and may lead to sophisticated compatibility problems. Consider excluding
-        //       these IDE packages from the plugin distribution. If your plugin depends on classes of an IDE bundled plugin, explicitly
-        //       specify dependency on that plugin instead of bundling it.
-        // IMPORTANT: Third party suppliers of rule set need to add those relocations as well!
-        relocate("org.jetbrains.org", "shadow.org.jetbrains.org")
-        relocate("org.jetbrains.concurrency", "shadow.org.jetbrains.concurrency")
+        relocate("org.jetbrains", "shadow.org.jetbrains")
+        relocate("org.intellij", "shadow.org.intellij")
 
         // Ktlint-lib itself may not be minimized as this would result in exceptions when loading custom rulesets as the RulesetProviderV3
         // can not be found

